frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-material-count/radiogram-card-content-material-count.component.ts

```diff
@@ -0,0 +1,39 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type {
+    MaterialCountRadiogram,
+    PatientStatus,
+} from 'digital-fuesim-manv-shared';
+import { UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import { map } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import { createSelectRadiogram } from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-radiogram-card-content-material-count',
+    templateUrl: './radiogram-card-content-material-count.component.html',
+    styleUrls: ['./radiogram-card-content-material-count.component.scss'],
+})
+export class RadiogramCardContentMaterialCountComponent implements OnInit {
+    @Input() radiogramId!: UUID;
+
+    capacity$!: Observable<Partial<{ [key in PatientStatus]: number }>>;
```

Comment:
I think you could use `Observable<{ red: number; yellow: number; green: number; }>` here, which would make it easier to understand for me